### PR TITLE
Update LeaseWeb.com.xml

### PR DESCRIPTION
**Pull request by wesdawg**

### CODE REVIEW

File: src/chrome/content/rules/LeaseWeb.xml [hunk 1]
```diff
@@ -9,23 +9,27 @@
...
 	<!--	Direct rewrites:
 				-->
 	<target host="leaseweb.com" />
-	<target host="secure.leaseweb.com" />
 	<target host="www.leaseweb.com" />
+	<target host="auth.leaseweb.com" />
+	<target host="mirror.leaseweb.com" />
+	<target host="secure.leaseweb.com" />
+	
+	<target host="*.leaseweb.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/LeaseWeb.xml]: This wildcard breaks various hosts.

Comment (wesdawg) [reply src/chrome/content/rules/LeaseWeb.xml]: How so ?

Comment (J0WI) [reply src/chrome/content/rules/LeaseWeb.xml]: Here are some:
```
ns7.leaseweb.net
ns8.leaseweb.net
www.leaseweb.net (same for ^ btw.)
www.mirror.de.leaseweb.net

Comment (J0WI) [reply src/chrome/content/rules/LeaseWeb.xml]: Can you add a comment for them on the top?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**